src/js/lib.js

```diff
@@ -16,22 +16,28 @@ var lib = (function() {
   }
 
   function errorLog(context) {
+    console.log('lib.errorLog deprecated.  Find a different way to pass this error.')
```

Comment:
If we are no longer using errorLog in `lib.js`, let's pull this out of the code.